Core/RELEASE-NOTES

```diff
@@ -4,6 +4,7 @@
 *Version 4.4.4*
 * Fix point values purge when data source and data point purge setting overrides disabled
 * Avoid loading all points into memory while purging
+* Added preformance update on cache for new points
```

Comment:
Typo - "performance". I think we should give a bit of a clearer description for this one.

Reply:
Maybe something like "Improved performance for new points by setting the cache to and empty collection as new points are not expected to have cache" ?

Reply:
"Improve performance for new points by setting the point value cache to an empty collection"